libminifi/test/unit/SchedulingAgentTests.cpp

```diff
@@ -63,8 +64,8 @@ class SchedulingAgentTestFixture {
     count_proc_->setScheduledState(core::RUNNING);
 
 #ifdef WIN32
-    utils::dateSetInstall(TZ_DATA_DIR);
     date::set_install(TZ_DATA_DIR);
+    minifi::utils::timeutils::dateSetInstall(TZ_DATA_DIR);
```

Comment:
Why was this reordered?

Reply:
It was removed, and readded so that's probably why the order changed, updated in 6c350a979f1fbdd0e2e020feeb0ad7087b64771c